### ISSUE TITLE
stylesheet 

### ISSUE THREAD

**Issue by djinnsynth** [r52/PTA#30]
Sorry for making this an "Issue" when actually it's just me being confused!

Having trouble getting Fontin as a the font your app uses. I THINK my stylesheet is correct but it could be wrong? Maybe if you had a guide to change it?

**Comment by djinnsynth** [comment]
Ok turns out you have to write:

"Fontin" and not "Fontin-Regular"

I'll show myself out . . .

**Comment by r52** [comment]
If you are referring to the styling within the `templates/price/index.html` file in the latest Release version of the app, I added the "Fontin-Regular" style early on hoping to emulate PoE's style but I never ended up shipping the font with the app. For most people that don't actually have the Fontin fonts installed on their system (including me), it is not a mistake to see Arial or Verdana as the font in the results UI.

If you are using the latest development version, I have since revamped and combined the advanced search and results dialogs into a single web UI that has "Fontin-SmallCaps" baked in. Unfortunately with this change, the ability to customize your own styles had to be removed as well.

Hope this helps.
